framework/src/wifi_manager/wifi_manager_state.c

```diff
@@ -684,6 +676,35 @@ wifi_manager_result_e _handler_on_scanning_state(wifimgr_msg_s *msg)
 	return wret;
 }
 
+wifi_manager_result_e _handler_get_stats(wifimgr_msg_s *msg)
+{
+	wifi_manager_result_e wret = WIFI_MANAGER_SUCCESS;
+	trwifi_msg_stats_s stats;
+	stats.cmd = TRWIFI_MSG_GET_STATS;
+	wifi_utils_result_e res = wifi_utils_ioctl(&stats);
+	if (res != WIFI_UTILS_SUCCESS) {
+		wret = WIFI_MANAGER_FAIL;
+	} else {
+		// update msg
+		wifi_manager_stats_s *wstats = (wifi_manager_stats_s *)msg->param;
+		wstats->start = stats.start;
```

Comment:
I think we'd better check `wstats` is NULL or not. It can be dereferencing.

Reply:
wstats is checked wifi_manager_result_e wifi_manager_get_stats(wifi_manager_stats_s *stats)
so it doesn't need to check again.